src/electron/windows/sign_windows_executable.mjs

```diff
@@ -98,6 +98,20 @@ function appendDigicertUsbJsignArgs(args, options) {
   args.push('--keystore', eTokenCfg);
```

Comment:
Sure we can, I'm thinking of a use case now cause for now it will only be invoked by `electron-builder`. But probably in the future we can use it to sign our service binaries in `tools`.

Reply:
Yeah, not blocking, just want to make sure we don't slide back into having multiple "build systems"

Reply:
updated to action. Also I fixed a small issue in `run_action`: when you run an action with argument containing spaces, `run_action` will expand it two multiple arguments because of the `{shell: true}` option set in `spawn`.

For example, `npm run action test -- param="abc def"` will be actual expanded to `node test.mjs param=abc def`. So i turned off the `{shell: true}` and put all subcommands in arguments list.

Reply:
mm, great, thanks!